science/nektarpp/Portfile

```diff
@@ -0,0 +1,162 @@
+description             spectral/hp element framework
+long_description        Nektar++ is a spectral/hp element framework which aims to be easy \
+                        to use and support a wide range of features, including 1D, 2D and \
+                        3D element support, hybrid shaped elements and continuous and \
+                        discontinuous implementations. As well as a modern C++ API, it \
+                        ships with a range of solvers including CFD, advection-diffusion \
+                        equations and cardiac electrophysiology.
+
+# Additional dependencies for Tetgen and Triangle
+master_sites-append     https://www.nektar.info/thirdparty/
+distfiles-append        tetgen-1.5.zip \
+                        triangle-1.6.zip
+
+checksums               nektar-5.0.0.tar.bz2 \
+                        rmd160  f588a4d6429b1ffc85be538885cbb09886ac9cba \
+                        sha256  d98c87af64e6d5cce04f8f60045a63fe63a79274c72a0ba368716fe603a4b625 \
+                        size    48486184 \
+                        tetgen-1.5.zip \
+                        rmd160  d8785f4ca6b26608ec9423f7574a0e736380370a \
+                        sha256  52207793198746de14abcb30a0aeed617d7348ff37544e7d7e65aaaa76d7fa70 \
+                        size    264128 \
+                        triangle-1.6.zip \
+                        rmd160  c92c011c442157cae5abcf246935d61bdfeede26 \
+                        sha256  7d74d7efbe35dc5a497bd179bdc87b4670740e466f67043d41e0b0546ce76cb5 \
+                        size    130331
+
+# Only the full Nektar++ source should be extracted - CMake will extract any
+# third-party dependencies.
+extract.only            nektar-5.0.0.tar.bz2
+
+patchfiles              no-homebrew.patch \
+                        avoid-python-install.patch \
+                        thirdparty-sources.patch
+
+compiler.cxx_standard   2011
+cmake.build_type        Release
+
+mpi.setup
+
+depends_lib-append      port:arpack \
+                        port:boost \
+                        port:fftw-3 \
+                        port:opencascade \
+                        port:scotch \
+                        port:tinyxml \
+                        port:zlib
+
+# Don't build demos and tests by default.
+configure.args-append   -DNEKTAR_BUILD_DEMOS=OFF \
+                        -DNEKTAR_BUILD_TESTS=OFF \
+                        -DNEKTAR_BUILD_UNIT_TESTS=OFF \
+                        -DNEKTAR_SOLVER_DIFFUSION=OFF \
+                        -DNEKTAR_SOLVER_DUMMY=OFF \
+                        -DNEKTAR_SOLVER_DIFFUSION=OFF \
+                        -DNEKTAR_SOLVER_ELASTICITY=OFF \
+                        -DNEKTAR_SOLVER_DIFFUSION=OFF \
+                        -DNEKTAR_SOLVER_MMF=OFF \
+                        -DNEKTAR_USE_ARPACK=ON \
+                        -DNEKTAR_USE_FFTW=ON \
+                        -DNEKTAR_USE_SCOTCH=ON \
+                        -DNEKTAR_USE_MESHGEN=ON
```

Comment:
It appears that there are duplicates in here. Can you sort them alphabetically and remove the dups?

Reply:
Okay have done!